src/gui/widgets/listbox.cpp

```diff
@@ -532,16 +532,29 @@ void listbox::finalize(builder_grid_const_ptr header,
 	if(header) {
 		swap_grid(&get_grid(), content_grid(), header->build(), "_header_grid");
 	}
+
 	grid& p = find_widget<grid>(this, "_header_grid", false);
+
 	for(unsigned i = 0, max = std::max(p.get_cols(), p.get_rows()); i < max; ++i) {
-		if(selectable_item* selectable = find_widget<selectable_item>(&p, "sort_" +  std::to_string(i), false, false)) {
-			selectable->set_callback_state_change(std::bind(&listbox::order_by_column, this, i, _1));
+		//
+		// TODO: I had to change this to case to a toggle_button in order to use a signal handler.
+		// Should probably look into a way to make it more general like it was before (used to be
+		// cast to selectable_item).
+		//
+		// vultraz - 8/23/2017
```

Comment:
This date format is ambiguous, and not all developers live in countries that use the M/D/Y date format. (Heck, the *only* notable countries which use MDY are USA, Micronesia, and Marshall Islands.)

Please specify the date as [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601): 2017-08-23.